Dev to main

DLBPointon opened the pull request:
Dev to Main

muffato commented on CHANGELOG.md:
```diff
@@ -3,6 +3,67 @@
+- Updating all modules.
+- Removing Anaconda 'defaults' channel.
+- Updating local module containers.
+- Update to LISCENSES and CITATIONS files.
```
```suggestion
- Update to LICENSE and CITATIONS files.
```

muffato commented on CHANGELOG.md:
```diff
@@ -3,6 +3,67 @@
+
+Even modules which have not had a version bump have indeed been updated through NF-core to remove defaults.
+
+Some modules now have two verions, the new addition is the script version rather than just the dependency version.
```
```suggestion
Some modules now have two versions, the new addition is the script version rather than just the dependency version.
```

DLBPointon commented:
The above changes you've requested are in #49

DLBPointon commented:
Thanks for the review, Matthieu.

The tag must have accidentally been on dev.

I've added the 1.0.1 changelog, there wasn't much, and updated the DOI in the above files.

There hasn't been any change to the fundamentals of the pipeline in terms of operation or flags.

Zenodo was updated manually.

muffato commented:
> Zenodo was updated manually.

Ah. The GitHub integration will create _another_ DOI because it doesn't know it needs to reuse the existing one.
Let me contact the Zenodo helpdesk see if we can force that. Otherwise we'll have to update the DOI again post release

muffato commented:
Hum, wait a sec. This is what I can see on the Zenodo interface:
<img width="940" alt="Screenshot 2024-12-13 at 17 17 14" src="https://github.com/user-attachments/assets/606a6e68-87d4-433a-85d0-a67511206fb3" />

which suggests that the existing uploads are recognised.

Let's just merge this in. In the worst case we'll change the DOI in the next release but even if that happens, existing links will still work
